src/sections/General/Navigation/utility/ScrollspyMenu.js

```diff
@@ -58,17 +101,17 @@ const ScrollspyMenu = ({ menuItems, ...props }) => {
       {active.subItems !== undefined && (
         <React.Fragment>
           <ul className="dropdown" style={{ zIndex: "101" }}>
-            <div className="nav-grid">
+            <div className="nav-grid" ref={dropdowncontent}>
               <div className="hr">
-                <Link className="section" to={active.path}  >{active.name}
+                <Link className="section" onKeyDown={handlefocus} to={active.path}  >{active.name}
                 </Link>
                 {active.subItems.map((subItem, i) => (
                   <li key={i}>
                     { subItem.name === "Forum" ?
-                      <a href={subItem.path} target="_blank" className="sub-item" rel="noreferrer">
+                      <a href={subItem.path} target="_blank" className="sub-item" rel="noreferrer" onKeyDown={handlefocus}>
```

Comment:
@Palaksharma23 What's your opinion on adding `aria-label` or `aria-describedby`?

This is a great reference for Accessible Anchor Links, https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/ which might help you answer the above question :)

Reply:
Yes, you are right we should be adding these! 
I will commit the necessary changes :)